src/sas/qtgui/Calculators/GenericScatteringCalculator.py

```diff
@@ -577,11 +625,23 @@ def change_is_avg(self):
             self.txtMx.setText("0.0")
             self.txtMy.setText("0.0")
             self.txtMz.setText("0.0")
+            self.txtQxMin.setEnabled(True)
+            self.txtQxMin.setText(str(float(self.txtQxMax.text())*.001))
+            self.checkboxLogSpace.setChecked(True)
+            self.checkboxLogSpace.setEnabled(True)
+            self.checkboxPluginModel.setEnabled(True)
         # If not averaging then re-enable the magnetic sld textboxes
         else:
             self.txtMx.setEnabled(True)
             self.txtMy.setEnabled(True)
             self.txtMz.setEnabled(True)
+            self.txtQxMin.setEnabled(False)
+            self.txtQxMin.setText(str(float(self.txtQxMax.text())*-1))
+            self.checkboxLogSpace.setChecked(False)
+            self.checkboxLogSpace.setEnabled(False)
+            self.checkboxPluginModel.setEnabled(False)
+            self.checkboxPluginModel.setChecked(False)
+            self.txtFileName.setEnabled(False)
```

Comment:
Most of these checked/enabled values are opposite depending on `self.is_avg` and each pair could be combined into a single line outside the if/else statement.

```
self.txtMx.setEnabled(not self.is_avg)
...
self.txtQxMin.setEnabled(self.is_avg)
...
multiple = 0.001 if self.is_avg else -1
self.txtQxMin.setText(str(float(self.txtQxMax.text())*mulitple)
...
etc.
```